coalib/results/AbsolutePosition.py

```diff
@@ -0,0 +1,59 @@
+from coalib.results.TextPosition import TextPosition
+from coalib.misc.Decorators import enforce_signature
+
+
+class AbsolutePosition(TextPosition):
+
+    @enforce_signature
+    def __init__(self,
+                 text: (tuple, list, None)=None,
+                 position: (int, None)=None):
+        """
+        Creates an AbsolutePosition object that represents the index of a
+        character in a string.
+
+        :param text:     The text containing the character.
+        :param position: Position identifying the index of character
+                         in text.
+        """
+        line = column = None
+        if position and text:
+            line, column = AbsolutePosition.calc_line_col(text, position)
+        self._text = text
+        self._position = position
+        super(AbsolutePosition, self).__init__(line, column)
+
+    @property
+    def position(self):
+        return self._position
+
+    @classmethod
+    def calc_line_col(cls, text, pos_to_find):
```

Comment:
why is this a classmethod? That doesn't make sense, it doesn't use `cls` and is pretty generic, should live outside, right?